modules/apps/portal-workflow/portal-workflow-task-web/src/test/java/com/liferay/portal/workflow/task/web/internal/notifications/WorkflowTaskUserNotificationHandlerTest.java

```diff
@@ -143,6 +147,84 @@ public void testInvalidWorkflowTaskIdShouldReturnBlankLink()
 				_serviceContext));
 	}
 
+	@Test
+	public void testIsApplicable() throws Exception {
+		_workflowTaskManager = Mockito.mock(WorkflowTaskManager.class);
+
+		ReflectionTestUtil.setFieldValue(
+			WorkflowTaskManagerUtil.class, "_workflowTaskManager",
+			_workflowTaskManager);
+
+		UserNotificationEvent userNotificationEvent = Mockito.mock(
+			UserNotificationEvent.class);
+
+		JSONObject jsonObject = JSONUtil.put(
+			"entryClassName", _VALID_ENTRY_CLASS_NAME
+		).put(
+			"workflowTaskId", _VALID_WORKFLOW_TASK_ID
+		);
+
+		Mockito.when(
+			userNotificationEvent.getPayload()
+		).thenReturn(
+			jsonObject.toJSONString()
+		);
+
+		DefaultWorkflowTask defaultWorkflowTask = new DefaultWorkflowTask();
```

Comment:
Hi @seltong

Could we need this object?

Reply:
No, we don't need